config/environments/production.rb

```diff
@@ -41,6 +40,9 @@
   # config.action_dispatch.x_sendfile_header = 'X-Sendfile' # for Apache
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
+  # Store uploaded files on the local file system (see config/storage.yml for options)
+  config.active_storage.service = :local
```

Comment:
This feels odd for production? Is this meant to be only for development?

Reply:
@shaunakpp, I believe you were working with Ian on the upgrade. Do you have insight into why this was added? And if it should be for production? If not, I can dig deeper. Just didn't want to chase answers down if you already had them. 😄

Reply:
It came up with the `app:update` task afaik, but we don't really use  active storage in classroom, yet, so we can comment it out I think.